command.go

```diff
@@ -0,0 +1,64 @@
+package task
+
+import (
+	"errors"
+	"strings"
+)
+
+type Params map[string]string
+
+type Cmd struct {
+	Cmd    string
+	Task   string
+	Params Params
```

Comment:
Based on what's easiest to document in the README.

If you need to rename "parms" -> "vars" for the end-user only, you could probably add a struct tag here and in dep, if your yaml lib supports it:
```go
  Params Params `yaml:"vars"`
```